File: Dia8/VolumenCilindro.js
//Volumen de un cilindro en JavaScript
const PI = 3.1416

class cylinder {
    constructor(radio, height) {
        this.radio = radio
        this.height = height
    }
    getArea() {
        return PI * Math.pow(this.radio, 2) * this.height
    }
}

const radio = parseFloat(prompt('Introduzca el radio de las bases del cilindro'))
const altura = parseFloat(prompt('Introduzca la altura del cilindro'))
alert(`El volumen del cilindro es ${new cylinder(radio, altura).getArea().toFixed(2)}`)

File: Dia08/VolumenCilindro.py
#Volumen de un cilindro en Python
PI = 3.1416

class cylinder():
    def __init__(self, radio, height):
        self.radio = radio
        self.height = height
    def getArea(self):
        return PI * self.radio**2 * self.height

radio = float(input('Introduzca el radio de las bases del cilindro: '))
altura = float(input('Introduzca la altura del cilindro: '))
print('El volumen del cilindro es: {}'.format(round(cylinder(radio, altura).getArea(), 2)))